tests/softlearning/environments/adapters/softlearning_env_test.py

```diff
@@ -18,7 +18,7 @@ def test_active_observation_shape(self):
         env = self.create_adapter()
         active_observation_shape = env.active_observation_shape
         self.assertEqual(len(active_observation_shape), 1)
-        self.assertIsInstance(active_observation_shape[0], int)
+        self.assertTrue(np.issubdtype(type(active_observation_shape[0]), np.int64))
```

Comment:
Should the second argument be `int` or `np.int`?

Reply:
It seems like `int`, `np.int`, and `np.int64` all work for both the dm_control tests and the softlearning tests. So I left it as `np.int64`.

Reply:
Let's make it `np.int`, I think that's a bit more general.